server/controllers/profile.js

```diff
@@ -0,0 +1,138 @@
+const Profile =  require('../models/Profile')
+const User = require('../models/User')
+const asyncHandler = require("express-async-handler");
+
+exports.createProfile = asyncHandler(async (req, res, next) => {
+
+const {firstName , lastName , description , gender , dateOfBirth , phone ,availability } = req.body
```

Comment:
```suggestion
const {firstName, lastName, description, gender, dateOfBirth, phone, availability } = req.body
```

Let's maintain proper spacing between our object destructuring.